app/experimenter/nimbus-ui/src/components/FormOverview/index.tsx

```diff
@@ -123,34 +127,37 @@ const FormOverview = ({
         <Form.Control
           {...nameValidated("hypothesis")}
           as="textarea"
-          rows={3}
-          defaultValue={experiment?.hypothesis || ""}
+          rows={5}
+          defaultValue={experiment?.hypothesis || hypothesisDefault || ""}
```

Comment:
I suppose this could just be `experiment?.hypothesis as string || hypothesisDefault`, no need for the empty string fallback